vembrane/__init__.py

```diff
@@ -49,52 +49,91 @@
 
 
 class Sample:
-    def __init__(self, record_idx: int, sample: str, format_data: Dict[str, Any]):
+    def __init__(self, record_idx: int, sample_name: str, sample):
```

Comment:
We should type annotate everything, `sample` is something along the lines of `Dict[str, whatever]`

Reply:
Yeah, this is `record.sample`.
I didn't look into this further because when I wanted to add `VariantRecordInfo` as a type I noticed it isn't exported by `pysam`...
Feel free to change this how you see fit!